src/components/Appointments/AppointmentDetails.vue

```diff
@@ -131,21 +134,15 @@ export default {
 		},
 	},
 	methods: {
-		async save() {
-			this.canClose = false
-			this.isLoading = true
-			try {
-				await this.$emit('save', {
```

Comment:
so the issue was that event listeners are not awaitable?

Reply:
yes